Documentation/spec/docfx_document_schema.md

```diff
@@ -137,8 +121,8 @@ It defines how applications interpret the property. If not defined, the behavior
 |------------|-------------
 | `default`  | It means that no interpretion will be done to the property.
 | `uid`      | `type` MUST be `string`. It means the property defines a unique identifier inside current document model
-| `fileLink` | `type` MUST be `string`. It means the property defines a file link inside current document model. Application CAN help to validate if the linked file exists, and update the file link if the linked file changes its output path.
-| `uidLink`  | `type` MUST be `string`. It means the property defines a UID link inside current document model. Application CAN help to validate if the linked UID exists, and resolve the UID link to the corresponding file output path.
+| `xref`     | `type` MUST be `string`. It means the property defines a file link inside current document model. Application CAN help to validate if the linked file exists, and update the file link if the linked file changes its output path.
+| `href`     | `type` MUST be `string`. It means the property defines a UID link inside current document model. Application CAN help to validate if the linked UID exists, and resolve the UID link to the corresponding file output path.
```

Comment:
question: should it in url encoding form?
should we add `file` for content without url encoding?
e.g. if a file name is `a%20b.md`, `href` should be `a%2520b.md`, `file` should be `a%20b.md`.